app/src/i18n/locales/de/auth.json

```diff
@@ -13,7 +13,6 @@
   "verified-toast-title": "Konto verifiziert!",
   "verified-toast-description": "Sie können sich nun mit Ihrem Konto anmelden.",
   "invalid-email-password": "Ungültige Email oder Passwort",
-
   "signup-heading": "Registrieren für City Catalyst",
   "signup-details": "Bitte tragen Sie ihre Details ein, um sich zu registrieren",
   "full-name": "Voller Name",
```

Comment:
Voller Name -> Vollständiger Name

Reply:
Awesome! I'm going to merge the changes, then add these updates.